### PR TITLE
makes documentation index page more concise

### PR DESCRIPTION
**Pull request by discdiver**
I have made the following changes 👍 
 made index page more helpful to potential users and users with issues

_If you've added to the docs_
- _Have you built the docs using `make html`?_
Yes

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -9,14 +9,49 @@ Yellowbrick: Machine Learning Visualization
...
+Thank you for your contributions!
+
+Concepts & API
+--------------
 
 Visualizers
 -----------
+We strive to maintain a brain-friendly API. Our API is similar to Scikit-Learn's API.
 
-Visualizers are estimators (objects that learn from data) whose primary objective is to create visualizations that allow insight into the model selection process. In Scikit-Learn terms, they can be similar to transformers when visualizing the data space or wrap an model estimator similar to how the "ModelCV" (e.g. `RidgeCV <http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.RidgeCV.html>`_, `LassoCV <http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LassoCV.html>`_) methods work. The primary goal of Yellowbrick is to create a sensical API similar to Scikit-Learn. Some of our most popular visualizers include:
```
Comment (wagner2010) [inline docs/index.rst]: We like these 2 sentences: "Visualizers are estimators..." and "The primary goal of Yellowbrick..." Perhaps we could them back in?  Thanks!

File: docs/index.rst [hunk 2]
```diff
@@ -76,27 +111,40 @@ Text Visualization
...
 - :doc:`api/text/postag`: plot the counts of different parts-of-speech throughout a tagged corpus
 
-... and more! Visualizers are being added all the time; be sure to check the examples (or even the `develop branch <https://github.com/DistrictDataLabs/yellowbrick/tree/develop>`_) and feel free to contribute your ideas for new Visualizers!
+... and more! Visualizers are being added all the time. Check the examples (or even the `develop branch <https://github.com/DistrictDataLabs/yellowbrick/tree/develop>`_). Feel free to contribute your ideas for new Visualizers!
 
 Getting Help
 ------------
 
-Yellowbrick is a welcoming, inclusive project in the tradition of matplotlib and scikit-learn. Similar to those projects, we follow the `Python Software Foundation Code of Conduct <http://www.python.org/psf/codeofconduct/>`_. Please don't hesitate to reach out to us for help or if you have any contributions or bugs to report!
+Can't get someting to work? Here are places to search for answers:
```
Comment (wagner2010) [inline docs/index.rst]: I really like that you put this into a list.  In the spirit of streamlining (a favorite activity of ours!), perhaps we could combine them to streamline it further?  IE
1.)The docs...
2.)Stackoverflow.  Note: if you can't find a solution, ask a question and tag it (also Reddit is growing in popularity too)...
3.)YB Google groups
4.)Tweet at us...

File: docs/index.rst (hunk 1 above)
Comment (discdiver) [reply docs/index.rst]: Sure.

File: docs/index.rst (hunk 2 above)
Comment (discdiver) [reply docs/index.rst]: Good thought. I've combined the lists.

**Pull request merged**